pkg/gui/recent_repos_panel.go

```diff
@@ -59,6 +59,24 @@ func (gui *Gui) updateRecentRepoList() error {
 	return gui.Config.SaveAppState()
 }
 
+// canShowIsPrivateRepo returns true if a private repo is never opend before in lazygit
+func (gui *Gui) canShowIsPrivateRepo() bool {
+	repos := gui.Config.GetAppState().RecentPrivateRepos
+	currentRepo, err := os.Getwd()
+	for _, repo := range repos {
+		if currentRepo == repo {
+			return false
+		}
+	}
+	if err != nil {
```

Comment:
looks like this error check should be happening above, right after the call to os.Getwd

Reply:
Fixed :)